docs/guides/signature-verification.md

```diff
@@ -0,0 +1,129 @@
+# Dasharo release signature verification
+
+Each published Dasharo release s signed with a signing key corresponding to
+given platform and versions. The key infrastructure is stored in
+[3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack).
+
+To verify the integrity of the binaries published in release notes on this
+site, please follow the instructions below:
+
+1. Import necessary keys to your keyring:
+
+    ```bash
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/keys/master-key/3mdeb-master-key.asc
+    gpg --import 3mdeb-master-key.asc
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/dasharo/3mdeb-dasharo-master-key.asc
+    gpg --import 3mdeb-dasharo-master-key.asc
+    wget <release signing key URL>
+    gpg --import <release signing key file>
+    gpg --list-sigs "3mdeb Master Key" "3mdeb Dasharo Master Key" <release signing key name>
+    ```
+
+    - `<release signing key URL>` is provided in the release notes
+    - `<release signing key file>` is the file name in the
+    - `<release signing key URL>`
+    - `<release signing key name>` will show up after importing the key via GPG.
```

Comment:
This description looks like unfinished and redundant

Reply:
https://github.com/Dasharo/docs/pull/621/files#r1277278084

Reply:
One had an extra dash